crates/fullnode/src/runner.rs

```diff
@@ -420,6 +423,13 @@ where
                     self.task_manager.abort().await;
                     return Ok(());
                 }
+                // Use this when SIGINT is not accepted
+                // Get the process id of the node and run `kill -15 <process_id>`
+                _ = shutdown_stream.recv() => {
+                    info!("Shutting down");
+                    self.task_manager.abort().await;
+                    return Ok(());
+                }
```

Comment:
This could have belonged to it's own PR since it's unrelated to the main context of the PR.

Reply:
Yeah, did not want to bother tbh :D

Reply:
Should I?

Reply:
yes please